aiida_ase/calculations/ase.py

```diff
@@ -36,10 +37,12 @@ def define(cls, spec):
         spec.input('kpoints', valid_type=KpointsData, required=False, help='The k-points to use for the calculation.')
         spec.input('parameters', valid_type=Dict, help='Input parameters for the namelists.')
         spec.input('settings', valid_type=Dict, required=False, help='Optional settings that control the plugin.')
+        spec.input('metadata.options.optimizer_stdout', valid_type=str, default=cls._OPTIMIZER_FILE_NAME,
+            help='Optimiser filename for relaxation')
 
-        spec.output('structure', valid_type=orm.StructureData, required=False)
-        spec.output('parameters', valid_type=orm.Dict, required=False)
-        spec.output('array', valid_type=orm.ArrayData, required=False)
+        spec.output('output_structure', valid_type=orm.StructureData, required=False)
+        spec.output('output_parameters', valid_type=orm.Dict, required=False)
+        spec.output('output_array', valid_type=orm.ArrayData, required=False)
```

Comment:
I think you may have added this change because the `output_` is common in other plugins. I am personally not a fan of this, since it is a bit redundant. For example when you want to get an output doing `node.outputs.structure` is just as clear as `node.outputs.output_structure`, if not clearer and you spare a lot of characters.

The only reason I didn't change it in other plugins I manage, is because it would break backwards compatibility. So I am wondering if we really should change this here?

Reply:
Yup absolutely, I did actually change it because I wanted it to look like the aiida-quantumespresso output. The other reason I changed it (for at least structure) was that I wanted to have an option later where I put a Trajectory file in with all the atoms leading up to the relaxed structure. I was hoping `output_structure` was clearly different and can be seen as the "final" structure. Same goes for the array, an output array would have everything pertaining to the last output. 

This can of course be done differently, so we just keep it to one word structure, parameters, trajectory, etc. I am fine with either and can change it back.

Reply:
The trajectory can always be added as a `TrajectoryData` (which as the name suggests is designed to hold structure trajectories) using the key `trajectory`. That is also what `aiida-quantumespresso` does for calculations that change the structure over multiple steps. Anyway, I think it might be best that (for now at least) you revert the change of the `output_` prefix, because it also breaks backwards compatibility. Admittedly, the number of users of this plugin might be low given that up to very recently it was not even compatible with AiiDA v1.0 and up. Still, such a decision would be best left to a separate PR as to not loose it between these other changes

Reply:
Yup absolutely, makes sense. Should be good now, reverted back to structure, parameters and array.